genai-perf/genai_perf/parser.py

```diff
@@ -818,8 +831,8 @@ def _add_input_args(parser):
         "should not have extensions. For example, "
         "'synthetic:queries,passages'. For payload data, prefix the path with 'payload:', "
         "followed by a JSON string representing a payload object. The payload should "
-        "contain fields such as 'timestamp', 'input_length', 'output_length', "
-        "and you can optionally add 'text_input', 'session_id', 'hash_ids', and 'priority'. "
+        "contain a 'timestamp' field "
```

Comment:
Yea okay. Seeing this typed out now, this is getting very verbose for a single option.
We likely want to link out to a tutorial here. @dyastremsky for viz

Reply:
@lkomali i think we make this a separate ticket
lets get this PR in, but we can add the doc in the next PR.